packages/test/mocha-test-setup/README.md

```diff
@@ -91,6 +95,24 @@ that to override them during execution time:
 FLUID_LOGGER_PROPS='{ "hostName": "Benchmark" }'
 ```
 
+### FLUID_TEST_LOGGER_PKG_SPECIFIER
```

Comment:
This behavior is unchanged in this PR, but the docs for it were missing and have been added. I don't know much about this option, but I figured minimal docs are better than none.